Resources/Prototypes/Entities/Objects/Specific/Robotics/mmi.yml

```diff
@@ -127,3 +131,7 @@
       guides:
       - Cyborgs
       - Robotics
+    - type: ChangeVoiceInContainer
+      whitelist:
+        components:
+          - SecretStash
```

Comment:
```suggestion
        components:
        - SecretStash
```

Reply:
ok I really need to change this from the pAI because that's where I'm copying it from 😭

Could I do that in this PR as well?

Reply:
> Could I do that in this PR as well?

I try to sneak stuff like that into my own PRs as well and others told me not to, so probably not 😞

Reply:
> I try to sneak stuff like that into my own PRs as well and others told me not to, so probably not 😞

Makes sense I guess. I'll remember next time I touch the pai stuff 😆